paddle/fluid/framework/operator.cc

```diff
@@ -34,6 +34,9 @@ DECLARE_bool(benchmark);
 DEFINE_bool(check_nan_inf, false,
             "Checking whether operator produce NAN/INF or not. It will be "
             "extremely slow so please use this flag wisely.");
+DEFINE_bool(check_multi_src_lods, false,
+            "For some operators with multiple inputs, check whether they have "
```

Comment:
"For some operators with multiple inputs, check whether they have "
            "identical LoDs."